### ISSUE TITLE
PyPi contains outdated version of pycricket

### ISSUE THREAD

**Issue by wolendranh** [roanuz/py-cricket#4]
Hi! 

It seems that version uploaded to PyPi is quite old(was uploaded in 2018). And actually does not work. 
Example code from github
```
import pycricket
handler = pycricket.RcaStorageHandler()
start = pycricket.RcaApp(access_key="Your_AccessKey", \
                        secret_key="Your_SecretKey", \
                        app_id="Your_APP_ID", \
                        store_handler=handler \
                  
```
Will fail with error 
`TypeError: argument of type 'bool' is not iterable`
Reason seems to be `a+` permission here:
``` 
    def read_file(self):
        """
        Open the file and assiging the permission to read/write and
        return the content in json formate.

        Return : json data
        """
        file_obj = open(self.file, 'a+')
```

**Comment by vinoth-rz** [comment]
Thanks for reporting @wolendranh, Sure, we will fix this problem.

**Comment by vinoth-rz** [comment]
@wolendranh Now we fixed this issue and updated the new version 1.0.10 👍
